typescript/github-proxy/tsconfig.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Auto generated by their CLI (wrangler)

Reply:
Need to dedupe probs

Reply:
Yes please de-dupe so we can ensure we're using consistent TS settings across the monorepo. Or at least as consistent as possible. If you need to have a few overrides in here that's fine

Reply:
fixed